File: backend/Data/Models/Enums/OrderStatus.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Data.Models.Enums
{
    public enum OrderStatus
    {
        CREATED,
        REJECTED,
        IN_PREPARATION,
        PICKED_UP,
        DELIVERING,
        COMPLETED,
    }
}

File: frontend/src/app/models/enums/order-status.ts
export enum OrderStatus
{
    CREATED = "CREATED",
    REJECTED = "REJECTED",
    IN_PREPARATION = "IN_PREPARATION",
    PICKED_UP = "PICKED_UP",
    DELIVERING = "DELIVERING",
    COMPLETED = "COMPLETED",
    READY="READY",
    PENDING="PENDING"
}

const orderStatus: Array<string> = Object.keys(OrderStatus).filter(key => isNaN(+key));

